.travis/run_tests.sh

```diff
@@ -21,7 +21,7 @@ if [ ! -z ${changed_files} ]; then
 		if [ ! -f ${f} ]; then
 			continue
 		fi
-		ts_name=$(grep -oP 'class[\s]+(\K\w+)(?=\(.*\):)' ${f})
+		ts_name=$(grep -oP 'class[\s]+(\KTest\w+)(?=\(.*\):)' ${f})
```

Comment:
This made me thinking that this grep will also select class names from __init__.py files which we don't want. So maybe try ignoring __init__.py file from the list of changed_files.